doc/whats_new.rst

```diff
@@ -78,6 +77,7 @@ API
 - Add ``datatype`` parameter and attribute to :class:`mne_bids.BIDSPath` that allows users to specify EEG, MEG, or iEEG datasets by `Adam Li`_ (`#514 <https://github.com/mne-tools/mne-bids/pull/514>`_)
 - Add ``datatype`` to replace ``kind`` parameter to :func:`mne_bids.make_bids_folders` and :func:`mne_bids.read_raw_bids` that allows users to specify EEG, MEG, or iEEG datasets by `Adam Li`_ (`#514 <https://github.com/mne-tools/mne-bids/pull/514>`_)
 - In :func:`mne_bids.write_raw_bids`, :func:`mne_bids.read_raw_bids`, :func:`mne_bids.get_matched_empty_room`, and :func:`mne_bids.get_head_mri_trans`, the ``bids_basename`` and ``bids_root`` keyword arguments have been removed. The functions now expect ``bids_path``, an instance of :class:`mne_bids.BIDSPath`, by `Adam Li`_ (`#525 <https://github.com/mne-tools/mne-bids/pull/525>`_)
+- :class:`mne_bids.BIDSPath` now has a function ``match``, which will get all matched basenames under the ``root`` directory by `Adam Li`_ (`#532 <https://github.com/mne-tools/mne-bids/pull/532>`_)
```

Comment:
suddenly we don't get credit for coming up with the regexp :)

Reply:
my bad! I'm working quickly just to get the CIs running :p. Let me modify.

Reply:
Updated! 

Feel free to modify accordingly :).